buildSrc/src/main/java/Testing.kt

```diff
@@ -11,7 +11,7 @@ object Testing {
     private const val truthVersion = "1.1.3"
     const val truth = "com.google.truth:truth:$truthVersion"
 
-    private const val mockkVersion = "1.10.0"
+    private const val mockkVersion = "1.11.0"
```

Comment:
I updated the version because of this issue.

https://github.com/philipplackner/CalorieTracker/issues/10